src/styles/custom.css

```diff
@@ -122,14 +117,28 @@
 	animation: 14s intro;
 }
 
+/* Tauri logo on index page */
+.hero > img {
+	animation: 3s intro;
+	opacity: 0.4;
+	width: 100%;
+}
```

Comment:
```suggestion
	animation: 3s intro;
	opacity: 0.4;
}
```

This was left over from the previous impl I believe and making it take up more screen space than needed on mobile.
Before: 
<img width="328" alt="Screenshot 2023-09-09 at 08 11 47" src="https://github.com/tauri-apps/tauri-docs/assets/15347255/cbf90a99-7b9f-495e-b851-92748b732e26">

After:
<img width="323" alt="Screenshot 2023-09-09 at 08 12 09" src="https://github.com/tauri-apps/tauri-docs/assets/15347255/7193dc44-4215-4947-90b2-913c4c7b2c6f">

Reply:
I had made the fix already but forgot to push it